content/en/blog/2023/tyk-api-gateway/index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Tyk API Gateway's Native OpenTelemetry Integration
+linkTitle: Tyk API Gateway's Native OpenTelemetry Integration
+date: 2023-10-24
+author: >-
+  [Sonja Chevre](https://github.com/SonjaChevre) (Tyk Technologies)
+draft: false
+---
+
+{{% alert color="primary" %}}
```

Comment:
The alert should probably have a title. How about something like "OTel community note" or "Integrations welcome"?